pyphare/pyphare/pharesee/run.py

```diff
@@ -371,10 +573,26 @@ def GetPi(self, time, merged=False, interp="nearest"):
         Pi = compute_hier_from(_compute_pressure, (M, massDensity, Vi))
         return self._get(Pi, time, merged, interp)
 
-    def GetJ(self, time, merged=False, interp="nearest"):
+    def GetPe(self, time, merged=False, interp="nearest", all_primal=False):
+        # TODO Te should come from the run... as an attribute of a hierarchy ?
+        Te = 1.0
+        hier = self._get_hierarchy(time, "ions_density.h5")
```

Comment:
with the latest PR merged putting the simulation in diagnostic files you should be able to change this to

```
        hier = self._get_hierarchy(time, "ions_density.h5")
        Te = hier.sim.electrons.closure.Te
```

Reply:
done